### PR TITLE
Update Dish: Add price field to dish

### PR DESCRIPTION
**Pull request by jaegarpoon**
Usage:
`adddish n/NAME $/PRICE`

Price should be >= 0.00 && <=99999.99

### CODE REVIEW

**Review by zxgoh — APPROVED** [review]
Okay!

File: src/main/java/seedu/address/model/dish/PriceDish.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+     * @param price A valid price.
+     */
+    public PriceDish(String price) {
+        requireNonNull(price);
+        checkArgument(isValidPrice(price), MESSAGE_CONSTRAINTS);
+        value = Double.parseDouble(price);
+    }
+
+    /**
+     * Returns true if a given string is a valid phone number.
```
Comment (xzzz3) [inline src/main/java/seedu/address/model/dish/PriceDish.java:30]: phone number -> price

**Review by xzzz3 — APPROVED** [review]
Looks great! Nice update of the package structure

**Comment by jaegarpoon** [comment]
Fixes #87

**Pull request merged**